README.md

```diff
@@ -8,6 +8,25 @@ Harvester is an open-source application that downloads data from various public
 [![Test Coverage](https://codeclimate.com/github/slovensko-digital/harvester.ecosystem/badges/coverage.svg)](https://codeclimate.com/github/slovensko-digital/harvester.ecosystem/coverage)
 [![Inch CI](https://inch-ci.org/github/slovensko-digital/harvester.ecosystem.svg)](https://inch-ci.org/github/slovensko-digital/harvester.ecosystem)
 
+## Development  
+
+Harvester is a Ruby-on-Rails application. However, there is no web process present. The application
+runs as a backend worker/scheduler and uses the Rails framework to enforce coding rules for the project. 
+
+For development, Harvester requires:
+
+ - Postgres 9.6+ and `psql` Postgres client `(to store scraped data) 
+ - Redis (for job queue management)
+
+Run the application as follows: 
+
+- Prepare the environment and start the worker with `bin/setup`
+- Ensure that all tests pass: `rake spec`
+- Run a selected job with `rake itms:all:sync`
```

Comment:
`itms:all:sync` actually enqueues a specific job (full ITMS2014+ sync). I'd recommend turning this into something more generic. Or just say that you can run harvester jobs with `rake`, it's a common practice in Ruby/Rails world.